### PR TITLE
set skipStmt using setter

### PR DESCRIPTION
**Pull request by kaavee315**
Changes to allow cqlproxy to run by allowing execution of statements not following prepare and execute paradigm. It can be specified in session if the user wants to skip the prepare statement and directly give command as string and execute

### CODE REVIEW

**Comment by kaavee315** [comment]
Checked the code with positive test case - migration tool, regression test - distributes_semaphore

File: session.go [hunk 1]
```diff
@@ -893,7 +908,7 @@ func (q *Query) PageState(state []byte) *Query {
 // CAS operations which do not end in Cas.
 //
 // See https://issues.apache.org/jira/browse/CASSANDRA-11099
-// https://github.com/gocql/gocql/issues/612
+// https://github.com/kaavee315/gocql/issues/612
```
Comment (garvitjuniwal) [inline session.go]: probably don't want to change this

File: session.go [hunk 2]
```diff
@@ -1570,7 +1585,7 @@ var (
 	ErrUnavailable          = errors.New("unavailable")
 	ErrUnsupported          = errors.New("feature not supported")
 	ErrTooManyStmts         = errors.New("too many statements")
-	ErrUseStmt              = errors.New("use statements aren't supported. Please see https://github.com/gocql/gocql for explanation.")
+	ErrUseStmt              = errors.New("use statements aren't supported. Please see https://github.com/kaavee315/gocql for explanation.")
```
Comment (garvitjuniwal) [inline session.go]: ditto

**Review by garvitjuniwal — APPROVED** [review]
LGTM

**Pull request merged**